### DISCUSSION TITLE
Kingston NV2 2TB works with Adapter

### DISCUSSION THREAD

**Discussion by FauthD** [R3NE07/Futro-S740#23] [Show and tell]
Hi, 
This NVMe SSD works just fine (one lane only) with that adapter:

https://de.aliexpress.com/item/1005004732439239.html?spm=a2g0o.productlist.main.5.5f897cd7tVWSiv&algo_pvid=da1ab7f0-3105-4bde-8179-967804f0027e&algo_exp_id=da1ab7f0-3105-4bde-8179-967804f0027e-2&pdp_npi=3%40dis%21EUR%215.44%214.19%21%21%21%21%21%402100baf316812101398377342d06d2%2112000030269985786%21sea%21DE%212467864453&curPageLogUid=gkpgX5Gfzf2r

Search: "
M2 TASTE EIN/E zu M2 NVME Adapter Karte M2 NVME zu KEY-M Expansion Slot WIFI interface zu M.2 SSD Festplatte Konverter T84C"

```
Filesystem      1K-blocks     Used  Available Use% Mounted on
tmpfs              790364     3220     787144   1% /run
/dev/nvme0n1p2   64237140 18854888   42046780  31% /
tmpfs             3951808        0    3951808   0% /dev/shm
tmpfs                5120        4       5116   1% /run/lock
/dev/nvme0n1p3 1855279628       60 1760963692   1% /srv
/dev/nvme0n1p1    1098628     6216    1092412   1% /boot/efi
tmpfs              790360       12     790348   1% /run/user/666
tmpfs              790360        8     790352   1% /run/user/0
```

LG, Dieter Fauth

**Comment by FauthD** [comment]
Forgot to mention it is connected to the A/E M.2 slot (the one normally used for WLAN etc).

Be aware of the mechanical length added due to the adapter.

**Comment by fly3rman** [comment]
I use the same adapter and here are some photos of the added length: https://www.dimitri-henning.de/projects/fujitsu_fustro_s740/

**Comment by cbs38139** [comment]
Out of curiosity: what file system did you use on both?

**Reply by fly3rman** [reply]
Ive had a little special setup: first M2 SATA ssd for Proxmox EXT4 + CEPH on the rest of the disk. And the second (the adapter nvme mentioned) also for CEPH. But thats interchangeable. I mostly use ext4 for system and BTRFS for data.

To further feed your curiosity and promote the flexibility of BTRFS:
I run an btrfs RAID10 (RAID1 earlier) on an another machine which constists of different sized:
- sata ssds
- nvme ssd
- usb3 -> nvme ssd
- usb3 -> ngff ssd

and looks like this:
```
btrfs fi show /mnt/diskblob
...
        Total devices 6 FS bytes used 5.00TiB
        devid    1 size 931.51GiB used 919.51GiB path /dev/sdb
        devid    4 size 1.82TiB used 1.79TiB path /dev/sdd
        devid    6 size 3.73TiB used 3.32TiB path /dev/sda
        devid    7 size 1.82TiB used 1.29TiB path /dev/sde
        devid    8 size 1.82TiB used 1.29TiB path /dev/sdg
        devid    9 size 1.86TiB used 1.80TiB path /dev/sdf
```

Its running stable after i found out how important good USB3 cables are :)))
Sorry for offtopic.
